cloud/logging.rst

```diff
@@ -43,26 +43,26 @@ Replace :code:`<token>` with the custom token listed in the provider's web site.
 
 **Papertrail**
 
-+------------------+-------------------------+
-| Hostname         | logs4.papertrailapp.com |
-+------------------+-------------------------+
-| Port             | 19493                   |
-+------------------+-------------------------+
-| TLS              | Yes                     |
-+------------------+-------------------------+
-| Protocol         | IETF Syslog             |
-+------------------+-------------------------+
-| Message Template |                         |
-+------------------+-------------------------+
++------------------+---------------------------+
+| Hostname         | logs<N>.papertrailapp.com |
```

Comment:
How should people determine the value for *N*? I had listed a specific value because it was what I had tested.

Reply:
It's given by papertrail. In my case it was `6`

Reply:
> In my case it was 6

I'm wondering whether we should suggest a particular number so that our users don't need to take extra steps to find a number, or whether that number is actually specific to each person's account, and must vary.

> papertrail has no authentication whatsoever

Papertrail leaves no papertrail. 😉

Reply:
I'm not sure. Maybe we could ask. If it's load balanced, e.g. paper trail customers are less than 30000-60000 ports or so, then I guess, though papertrail may not like the pile-up. If it's part of the routing, then it won't work.

Thing is, if you look at it, the host:port is the *only disambiguating information they have* about who is who.